betatest/templates/betatest/reject_email.txt

```diff
@@ -0,0 +1,10 @@
+{% autoescape off %}
+Hi {{ application.user.profile.full_name }},
+
+we are sorry to inform you that after careful consideration we were unable to
```

Comment:
In German, we use a lower-case letter after the comma in the salutation, but you are right, in English it seems to be more common to use an upper-case letter here. :)